### ISSUE TITLE
Are the backbones in provided baseline models fine-tuned? 

### ISSUE THREAD

**Issue by ziyanyang** [facebookresearch/detr#236]
Hi, I check the provided baseline model DETR-DC5 | R101, but its backbone resnet101 has the same parameters as pre-trained resnet101 from torchvision. Does it mean that such backbone resnet101 is not trained on COCO? But the paper mentioned they set learning rate in the transformer to 1e-4 and 1e-5 in the backbone.

**Comment by fmassa** [comment]
Yes, the backbones are trained.

We follow standard practice in Faster R-CNN models, and keep the first resnet block frozen, and fine-tune the other 3 blocks, see https://github.com/facebookresearch/detr/blob/5e66b4cd15b2b182da347103dd16578d28b49d69/models/backbone.py#L63-L64

I believe I've answered your question, and as such I'm closing this issue, but let us know if you have further questions
